setup.cfg

```diff
@@ -33,8 +33,8 @@ filterwarnings =
     ignore::ImportWarning
     ignore:::grpcio[.*]
     ignore:::google[.*]
-    # boto imports imp not importlib, but boto3 update not planned, see 1433148
-    ignore:.*the imp module is deprecated:DeprecationWarning
+    # boto3 kicks up ResourceWarning: unclosed socket, https://github.com/boto/boto3/issues/454
+    ignore:unclosed:ResourceWarning
```

Comment:
However, boto3 kicks up a `ResourceWarning`, so we have to ignore that in the tests otherwise it prints a bunch of stuff out that's annoying and unhelpful.